src/Accessibility/AXHearingUtilities.cs

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright 2021 Microsoft Corp
+//
+// Authors:
+//   Rachel Kang (rachelkang@microsoft.com)
+//
+
+#nullable enable
+
+#if !TVOS && !MONOMAC
+
+using System;
+using System.Runtime.InteropServices;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace Accessibility {
+    public static partial class AXHearingUtilities {
+
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
+        [DllImport (Constants.AccessibilityLibrary)]
+        static extern /* AXHearingDeviceEar */ nint AXMFiHearingDeviceStreamingEar ();
+
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
+        [DllImport (Constants.AccessibilityLibrary, EntryPoint = "AXSupportsBidirectionalAXMFiHearingDeviceStreaming")]
+        [return: MarshalAs (UnmanagedType.I1)]
+        static extern bool SupportsBidirectionalStreaming ();
```

Comment:
If you don't make it public, nobody will be able to use it!

```suggestion
        public static extern bool SupportsBidirectionalStreaming ();
```

Reply:
oops, thank you!